src/main/kotlin/com/grepp/nbe1_3_team04/team/domain/Team.kt

```diff
@@ -0,0 +1,71 @@
+package com.grepp.nbe1_3_team04.team.domain
+
+import com.grepp.nbe1_3_team04.global.domain.BaseEntity
+import jakarta.persistence.*
+import org.hibernate.annotations.SQLDelete
+
+
+@SQLDelete(sql = "UPDATE team SET is_deleted = 'TRUE' WHERE team_id = ?")
+@Entity
+class Team (
```

Comment:
정적 팩토리 메서드로 객체를 생성하니 주 생성자는 막아주시면 됩니다!
```suggestion
class Team private constructor (
```

Reply:
다른 클래스에도 적용하겠습니다!
감사합니다!